Update dependencies

davideme opened the pull request:
- Update Gradle Version
- Update kotlin version
- Update shot version
- Update AndroidX dependencies
- Update annotationProcessorOptions for room
- Disable blinking cursor Thanks @pedrovgs 
PS: Yes, constraintlayout is still in beta :(

davideme committed:
Update dependencies

- Update Gradle Version
- Update kotlin version
- Update shot version
- Update AndroidX dependecies
- Update annotationProcessorOptions for room

davideme committed:
Update dependencies

- Update work
- Update navigation

davideme committed:
update recordings

davideme committed:
Update .travis.yml

davideme committed:
Update .travis.yml

davideme committed:
upgrade simulator to api 27

davideme committed:
Update .travis.yml

davideme committed:
Update .travis.yml

davideme committed:
Update .travis.yml

davideme committed:
Update .travis.yml

davideme committed:
change SDK to 26

davideme committed:
update SDK and config.ini

davideme committed:
Update config.ini

davideme committed:
Delete .travis.yml

davideme committed:
Update create_emulator.sh

davideme committed:
Update Screenshots

davideme committed:
Update Screen size

pedrovgs committed:
Disable cursor from EditSuperHeroActivity if we are running UI tests

davideme committed:
Update config.ini

davideme committed:
Update screenshot size

davideme committed:
Merge branch '00-start-update' of https://github.com/Karumi/KataSuperHeroesJetpack into 00-start-update

Serchinastico commented on app/src/main/java/com/karumi/jetpack/superheroes/ui/view/EditSuperHeroActivity.kt:
```diff
@@ -92,4 +96,15 @@ class EditSuperHeroActivity : BaseActivity(), EditSuperHeroPresenter.View {
+
+    private fun disableEditTextCursorIfNeeded() {
+        if (TestConfig.runningTests) {
+            try {
```
I guess calling `setCursorVisible(false)` didn't work? 🤔

davideme committed:
isCursorVisible = fasle

Serchinastico committed:
Hide scrollbars in super hero detail

I'm removing the scrollbar hints in the super hero detail for two
reasons:
1. They are not used anymore in modern mobile design, users no longer
   understand that pattern out of web pages.
2. It's causing poblems in our screenshot tests. For some reason, they
   appear in a non-deterministic way while capturing screenshots.
